### PR TITLE
[ENG-1550] Admin app institutional group management.

### PR DESCRIPTION
**Pull request by corbinSanders**
## Purpose

The admin app will need the functionality to manage the new institutional permission groups so some institutional users can view the new metrics.

## Changes

Adding a new view, form, and html template to support this functionality.

## QA Notes

    - Is this an additive or subtractive change, other?
Additive change
  - How can QA verify? (Through UI, API, AdminApp or AdminAdminApp?)
This might need to be devtested as we need to ensure users are added successfully to permission groups

## Documentation

N/A

## Side Effects

There shouldn't be 

## Ticket

https://openscience.atlassian.net/browse/ENG-1550

### CODE REVIEW

**Comment by brianjgeiger** [comment]
Looks like there's still a migration problem on this.

**Comment by brianjgeiger** [comment]
Could just be a caching issue. I cleared the cache and restarted the tests.

File: admin/templates/institutions/detail.html [hunk 1]
```diff
@@ -50,6 +50,9 @@ <h4>Banner:</h4>
                     <button id="show-modify-form" class="btn btn-link" type="button">
                         Modify Institution
                     </button>
+                    <div class="col-md-12">
+                        <a class="btn btn-link" href={% url 'institutions:register_metrics_admin' institution.id %}>Register Moderator/Admin</a>
```
Comment (Johnetordoff) [inline admin/templates/institutions/detail.html]: I'd say "create Moderator/Admin" usually we only say register when making new accounts.

File: admin/templates/institutions/register_institutional_admin.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            </ul>
+            {% endif %}
+            <div class="form-group">
+                <label for="email">OSF ID:</label>
+                <p>{{ form.user_id }}</p>
+                {% if form.user_id.errors %}<span class="text-danger"> {{ form.user_id.errors }}</span>{% endif %}
+            </div>
+            <div class="row">
+                <div class="col-xs-4">
+                    <button type="submit" class="btn btn-primary btn-block btn-flat">Register</button>
```
Comment (Johnetordoff) [inline admin/templates/institutions/register_institutional_admin.html]: Here too, we aren't technically registering.

**Pull request merged**